### PR TITLE
Allow adding Sauce Labs in the My-Account.

### PR DESCRIPTION
**Pull request by nedSaf**
#131

### CODE REVIEW

**Comment by nedSaf** [comment]
@amitaibu The two accounts open and close together, Is that ok ?
account

**Comment by nedSaf** [comment]
Backend:
nader77___shoov

File: client/app/scripts/controllers/account.js [hunk 1]
```diff
@@ -17,12 +17,22 @@ angular.module('clientApp')
     Account.get().then(function(val) {
       $scope.account.browserstack_username = val.browserstack_username;
       $scope.account.browserstack_key = val.browserstack_key;
+      $scope.account.saucelabs_username = val.saucelabs_username;
```
Comment (amitaibu) [inline client/app/scripts/controllers/account.js]: saucelabs_username => sauce_username
saucelabs_key => sauce_access_key

Comment (amitaibu) [inline client/app/scripts/controllers/account.js]: (this is because these are name of the keys Sauce Labs are using)

Comment (nedSaf) [inline client/app/scripts/controllers/account.js]: Change fields names in Drupal as well ?

Comment (amitaibu) [inline client/app/scripts/controllers/account.js]: no, keep as is in Drupal.

**Comment by nedSaf** [comment]
@amitaibu Changed form behaviour and names:
account

**Comment by amitaibu** [comment]
Merged, thanks.

**Pull request closed without merging**